Automattic/wp-calypso
# A4A: add stats upsell for non jetpack sites

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/471

## Proposed Changes

Adding possibility to Add Jetpack stats when we don't have jetpack plugin installed.
This change will appear on Sites Dashboard for non-Atomic and sites with no Jetpack plugin.

Feel free to advice on style update and/or reuse of existing components. I couldn't use `<JetpackProductCard />` because of many changes required.

https://github.com/Automattic/wp-calypso/assets/60262784/6c1c4268-5911-46fe-b212-7fa97f885fba


## Why are these changes being made?


## Testing Instructions



- Have a site that is connected through Jetpack and at least one that only connected via A4A plugin
- Using live link below, navigate to `/sites` page and check stats column.
- You should see `+Add` button for sites without Jetpack plugin.
- Click the button and check the upsell.

## Pre-merge Checklist
